### DISCUSSION TITLE
[Feature] Menu can be scroll, for more simplicity

### DISCUSSION THREAD

**Discussion by spatiu** [HEIGE-PCloud/DoIt#160] [Ideas]
### Hello, I tried to implement it myself, but failed and that's why I ask you to implement it and the menu I think will look much better.

In present menu is in this way " Posts Tags Categories Series Authors Docs About " is nice i like it , but for me isnt good because i will have a lots of menu,   and to fix all the menus I thought to turn the menu into scroll menu but I don't know how to implement this, here is an example
I tried to implement it myself but I succeeded and that's why I ask you to implement it, and it will look much nicer..

I just want have only "1" menu, where i click on "name MENU" it scroll down all other menus (for exemple) all menu " Posts Tags Categories Series Authors Docs About"   as exemple in pics with LInks ( Menu) and scroll down .

![image](https://user-images.githubusercontent.com/74497946/126908352-6927047b-f595-4963-8894-29b8f884b051.png)
the exemple pics is take from theme clarity > https://github.com/chipzoller/hugo-clarity 

This way I think it looks much nicer and simpler and much more efficient because I can have many more menus with longer names.

**Comment by HEIGE-PCloud** [comment]
My suggestion for such need is to create a dedicate menu post for your website (similar to the [about page](https://hugodoit.pages.dev/about/) but with links to different menus inside).

I understand what you want is some drop-down menu like this

![image](https://user-images.githubusercontent.com/52968553/126948118-4d68dd70-1e82-4e7a-9ddd-91557e6b3dd5.png)

which is quite complicate to organize and implement, and not a common need for general personal use cases.

**Reply by spatiu** [reply]
Hi , thank for reply. 
I was thinking of using   [https://hugodoit.pages.dev/showcase-tests/](url) Showcase for create a menu , but your sugestion is better make as i see in about, where you inserted all License. + i think if i write menu in markdown with use of ### ## etc is more simple and effective . Thanks for sugestion . i just ask in first place if you can make scroll menu  if it would be easy to implement this feature. dont matter now because your sugestion is better for me . Thanks again :+1:  :smile:
